src/app/pages/BuyPage/components/BondingCurve/index.module.scss

```diff
@@ -0,0 +1,82 @@
+// swap btn
+.swap-btn-container {
+  max-width: 250px;
+  margin: auto;
+  margin-top: 23px;
+
+  .swap-btn-helper {
+    @apply tw-flex tw-items-center tw-justify-center
+      tw-text-lg tw-text-black tw-text-center tw-tracking-normal;
+
+    img {
+      @apply tw-ml-2 tw-cursor-pointer;
+    }
+  }
+
+  .swap-btn {
+    @apply tw-flex tw-items-center tw-justify-center
+      tw-p-0 tw-mx-auto tw-mt-4
+      tw-text-xl tw-font-bold
+      tw-bg-trade-long tw-rounded-lg;
+    height: 60px;
+    width: 250px;
+    padding: 0;
+  }
+}
+
+// swap form
+.swap-form-container {
+  display: flex;
```

Comment:
Pls update the styles to use `tailwind css` as much as possible.

Reply:
fixed.